ONBOARD.md

```diff
@@ -0,0 +1,104 @@
+## Prepare configuration for component repo
+
+The JSON configuration file describes how to build different images in a
```

Comment:
Add a few sentences to this paragraph describing how these are instructions for adding new CI jobs to the OpenShift CI systems.